docs/source/guide/helm_values.md

```diff
@@ -34,166 +34,173 @@ meta_description: For cases when you want to customize your Label Studio Enterpr
 | `rqworker.enabled`                                           | string   | true     | Enable rqworker pod                                                                         |
-| `rqworker.NameOverride`                                      | string   | ""       | String to partially override release template name                                          |
-| `rqworker.FullnameOverride`                                  | string   | ""       | String to fully override release template name                                              |
+| `rqworker.NameOverride`                                      | string   | true       | String to partially override release template name                                          |
+| `rqworker.FullnameOverride`                                  | string   | true       | String to fully override release template name                                              |
 | `rqworker.deploymentStrategy.type`                           | string   | Recreate | Deployment strategy type                                                                    |
 | `rqworker.replicas`                                          | string   | 1        | Amount of rqworker replicas                                                                 |
 | `rqworker.resources.requests.memory`                         | string   | 256Mi    | The requested memory resources for the Rqworker container                                   |
-| `rqworker.resources.requests.cpu`                            | string   | 100m     | The requested cpu resources for the Rqworker container                                      |
+| `rqworker.resources.requests.cpu`                            | string   | 100m     | The requested CPU resources for the Rqworker container                                      |
 | `rqworker.resources.limits.memory`                           | string   | 512Mi    | The memory resources limits for the Rqworker container                                      |
-| `rqworker.resources.limits.cpu`                              | string   | 500m     | The cpu resources limits for the Rqworker container                                         |
+| `rqworker.resources.limits.cpu`                              | string   | 500m     | The CPU resources limits for the Rqworker container                                         |
 | `rqworker.logLevel`                                          | string   | INFO     | Rqworker log level                                                                          |
-| `rqworker.debug`                                             | string   | ""       | Rqworker DEBUG mode                                                                         |
+| `rqworker.debug`                                             | string   | INFO        | Rqworker DEBUG mode                                                                         |
```

Comment:
not sure about "INFO", maybe there should be `1`..

Reply:
Yes as per the docs page.